hw/xbox/nv2a/nv2a_debug.c

```diff
@@ -57,13 +57,14 @@ void gl_debug_initialize(void)
 
 void gl_debug_message(bool cc, const char *fmt, ...)
 {
+    size_t n;
+    char buffer[1024];
+    va_list ap;
```

Comment:
Personal opinion: I prefer variable declarations where they are used. This legacy "everything on top" is often leading to much scrolling to check types in my opinion. Anyhow, fine in this instance, I just wanted to whine a little.